qiskit/synthesis/qft/qft_decompose_full.py

```diff
@@ -0,0 +1,61 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Circuit synthesis for a QFT circuit.
+"""
+
+from typing import Optional
+import numpy as np
+from qiskit.circuit.quantumcircuit import QuantumCircuit
+
+
+def synth_qft_full(
+    num_qubits: int,
+    do_swaps: bool = True,
+    approximation_degree: int = 0,
+    insert_barriers: bool = False,
+    inverse: bool = False,
+    name: Optional[str] = None,
+) -> QuantumCircuit:
+    """Construct a QFT circuit using all-to-all connectivity.
+
+    Args:
+        num_qubits: The number of qubits on which the QFT acts.
+        do_swaps: Whether to include the final swaps in the QFT.
```

Comment:
perhaps it's worth to explain this parameter better, and that it actually means "reverse qubits"
(also in `qft_decompose_lnn` function)

Reply:
Done in [0d06898](https://github.com/Qiskit/qiskit/pull/11463/commits/0d0689807a359d0172a7412a9e46b1c6fb626401). I think I have explained much better what ``do_swaps`` stands for (in both synthesis functions). The name of this argument is terrible, but changing it via our deprecation policy is even more terrible.